packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/Reflow/SimplePanning.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Button from 'terra-button/lib/Button';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/src/components/Bar/BarGraph.module.scss';
+import getBarPanningConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Bar/barPanning';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Bar/simplePanningData';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+
+const initialState = {
+  initial: 0,
+  factor: 3,
+};
+
+const graphConfig = utils.deepClone(getBarPanningConfig('#simpleBarPanning'));
+const dataset = utils.deepClone(data[0]);
+
+let graph;
+
+const BarPanningExample = () => {
+  React.useEffect(() => {
+    graph = Carbon.api.graph(graphConfig);
+    graph.loadContent(Carbon.api.bar(dataset));
+  }, []);
+
+  const reducer = (panState, action) => {
+    let hour;
+
+    switch (action.type) {
+      case 'panLeft':
+        hour = panState.initial - panState.factor;
+        break;
+      case 'panRight':
+        hour = panState.initial + panState.factor;
+        break;
+      default:
+        return panState;
+    }
+
+    graph.config.axis.x.lowerLimit = new Date(2016, 0, 1, hour).toISOString();
+    graph.config.axis.x.upperLimit = new Date(2016, 0, 2, hour).toISOString();
```

Comment:
See my related comment regarding side-effects inside reducer functions: https://github.com/cerner/terra-graphs/pull/193#discussion_r679991504

https://codeartistry.io/what-is-a-reducer-in-javascript-f4fd40f98324

Reply:
Fixed here: 8b9a77697c7fcea954b92104e778a7f742398301
(the no newline error was fixed in a subsequent commit)